Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -1263,6 +1269,14 @@ theorem sup_eq_sup {ι ι' : Type u} (r : ι → ι → Prop) (r' : ι' → ι'
     sup.{_, v} (familyOfBFamily' r ho f) = sup.{_, v} (familyOfBFamily' r' ho' f) :=
   sup_eq_of_range_eq.{u, u, v} (by simp)
 
+theorem IsNormal.iSup {f : Ordinal.{u} → Ordinal.{max u v}} (H : IsNormal f) {ι : Type w}
+    (g : ι → Ordinal.{u}) (hg : BddAbove (range g)) [Nonempty ι] : f (iSup g) = iSup (f ∘ g) :=
```

Comment:
This already exists as `IsNormal.map_iSup`. I think it would be good to have two versions:
- One taking in a `BddAbove` argument as this one.
- One that infers it whenever the domain is a small type.